data/crac-creation/crac-creator-csa-profiles/src/test/java/com/farao_community/farao/data/crac_creation/creator/csa_profile/crac_creator/CsaProfileCracUtilsTest.java

```diff
@@ -0,0 +1,50 @@
+package com.farao_community.farao.data.crac_creation.creator.csa_profile.crac_creator;
```

Comment:
is it deliberate not to include a header ?

Reply:
done